examples/example-generation/examples/gp.sg

```diff
@@ -0,0 +1,56 @@
+# Importing generated data
+CSVFile X("../data/gaussian_process_X.csv")
+CSVFile X_test("../data/gaussian_process_X_test.csv")
+CSVFile Y("../data/gaussian_process_Y.csv")
+CSVFile Y_test("../data/gaussian_process_Y_test.csv")
+
+RegressionLabels labels(Y)
+RegressionLabels labels_test(Y_test)
+RealFeatures feats_train(X)
+RealFeatures feats_test(X_test)
+
+# GP specification
+GaussianKernel kernel(10, 2)
```

Comment:
We could support statements like that but if you are suggesting that the meta-language should actually look up the parameter names in the Shogun API in order to get the positions of the named arguments, that would complicate things quite a bit (a lot actually).
Furthermore, when the examples are translated these kinds of named arguments would be lost in all target languages that do not support this feature.

Say you have a meta-language example with the line (supposing the meta-language now supports this kind of statement):

```
GaussianKernel kernel(size=10, width=2)
```
Then this could potentially be translated to a python program as:

```
kernel = GaussianKernel(size=10, width=2)
```

But in Java this would just be translated to:

```
GaussianKernel kernel = new GaussianKernel(2, 10);
```

So for many target languages this wouldn't be very useful (keep in mind that most users will only look at the translations, not the meta-language versions). Another approach is to simply write the meta-language examples in a more explanatory way. E.g.

```
int size = 10
int width = 2
GaussianKernel kernel(size, width)
```
Which would translate well to all target languages.